src/css/components/mobile-tablet-menu.css

```diff
@@ -0,0 +1,128 @@
+/* Initial hidden state for overlay and menu */
+.menu-overlay {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.5s ease, visibility 0.5s ease;
+}
+
+/* Shown state with is-shown class */
```

Comment:
Please remove comments, because we write styles in this file only for one feature.